pkg/webhook/server/certificate/configuration.go

```diff
@@ -109,5 +109,10 @@ func (m *Manager) CABundle() ([]byte, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to get %s webhook configuration %s", m.webhookType, m.webhookName)
 	}
-	return m.clientConfigList(webhook)[0].CABundle, nil
+
+	if clientConfigList := m.clientConfigList(webhook); len(clientConfigList) == 0 {
```

Comment:
else + returns is a "no-no"

Something like this is more idiomatic with golang and with the rest of the code here.
```golang
clientConfigList := m.clientConfigList(webhook)
if len(clientConfigList) == 0 {
   return nil, errors.Wrapf(err, "failed to get clientConfig in %s webhook configuration %s", m.webhookType, m.webhookName)
}

return clientConfigList[0].CABundle, nil
```

Reply:
ack